app/controllers/api/search_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::SearchController < Api::ApiController
   def search
     @current_page = [params[:page].to_i, 1].max
     @query = params[:q]
-    @filters = params[:filters]
+    @filters = params[:filters]&.permit(:author, :thread, :subforum)&.to_h
```

Comment:
Needs `to_h` because `ActionController::Parameters` is no longer a hash, and we use `Hash#map` in `Post.generate_query`. Could have used `to_unsafe_h` but I thought there might have been a security issue in that we allowed people to pass anything to the filter list. I could only get author to work in the filter list. I think it's possible we no longer use thread and subforum (we jump right to the thread or subforum in the suggestion list).